### PR TITLE
add multivariate anomaly detetor sample for v1.1Dpg anomalydetector

### PR DESCRIPTION
**Pull request by happyling123**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
+
+require: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AnomalyDetector/readme.md
 output-folder: ../
+java: true
+regenerate-pom: false
+data-plane: true
+generate-tests: true
+artifact-id: azure-ai-anomalydetector
+generate-samples: true
+license-header: MICROSOFT_MIT_NO_CODEGEN
```
Comment (weidongxu-microsoft) [inline sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md:14]: license-header: MICROSOFT_MIT_SMALL

File: sdk/anomalydetector/azure-ai-anomalydetector/README.md [hunk 2]
```diff
@@ -59,43 +59,6 @@ You will also need to [register a new AAD application][register_aad_app] and [gr
 
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
 
-##### Async client
```
[the remark this answers is not in the document]
Comment (weidongxu-microsoft) [reply sdk/anomalydetector/azure-ai-anomalydetector/README.md:62]: Your recorded test is still on v1.1-preview.1. https://github.com/Azure/azure-sdk-for-java/blob/d10f0fa8325a311437e299be97a2cecc459b4e88/sdk/anomalydetector/azure-ai-anomalydetector/src/test/resources/session-records/testDetect.json#L4

Apparently it won't work. New recording is required https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods-Quickstart-with-AutoRest#recordplayback-tests

Refer to https://github.com/Azure/azure-sdk-for-java/blob/d10f0fa8325a311437e299be97a2cecc459b4e88/sdk/anomalydetector/azure-ai-anomalydetector/src/test/java/com/azure/ai/anomalydetector/generated/AnomalyDetectorClientTestBase.java for proper initialization of the client for record/playback test.

**Review by weidongxu-microsoft — COMMENTED** [review]
CHANGELOG needs update as well, you can refer to what you input for other languages.
https://azure.github.io/azure-sdk/policies_releases.html#change-logs

File: sdk/anomalydetector/azure-ai-anomalydetector/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (minkefusiji) [reply sdk/anomalydetector/azure-ai-anomalydetector/README.md:62]: @jr-MS  please help add sample in readme.
@happyling123 please help update recording in record/playback test.

**Comment by weidongxu-microsoft** [comment]
You swagger and Cadl https://github.com/Azure/azure-rest-api-specs/pull/21525 seems having one API named differently. Suggest for your internal confirmation.
https://apiview.dev/Assemblies/Review/077220369a574080afd4df85bd0ed871?diffRevisionId=05056228c2224452b71554d109e908b9&diffOnly=False&revisionId=9b17e22bae854f3689f18da80fc0dbac&doc=False

File: sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md (hunk 1 above)
Comment (happyling123) [reply sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md]: updated.

**Comment by happyling123** [comment]
I have resolved all the above comments, including:
- update CHANGELOG.
- add README sample.
- update recording in record/playback test.

**Pull request closed without merging**